build/build.go

```diff
@@ -1,6 +1,8 @@
 package build
 
 var (
-	Name    = "treefmt"
+	//nolint:gochecknoglobals
+	Name = "treefmt"
+	//nolint:gochecknoglobals
 	Version = "v0.0.1+dev"
```

Comment:
Does this affect only the next line? https://golangci-lint.run/usage/false-positives/#nolint-directive says this is "to exclude issues for the block of code". I'm not sure what "block of code" means here. Is it the entire `var(...)` directive?

I personally would write these as inline hints, which the docs clearly say "excludes issues only for this line."

```suggestion
	Name = "treefmt" //nolint:gochecknoglobals
	Version = "v0.0.1+dev" //nolint:gochecknoglobals
```